config/explorer.php

```diff
@@ -20,6 +20,8 @@
 
     'migration_address'                 => env('ARKSCAN_MIGRATION_ADDRESS'),
 
+    'polygonscan_url'                   => env('POLYGONSCAN_BASE_URL', 'https://polygonscan.com'),
```

Comment:
we could move it to the network type definitions, so we can default to different URLs based on the network type. That way the defaults will be usable for mainnet and devnet explorers already.